continuous-deployment/secrets-management/recipes/azure-devops/secrets-per-branch.md

```diff
@@ -1,14 +1,12 @@
-## Azure DevOps: managing settings on a per-branch basis
+# Azure DevOps: managing settings on a per-branch basis
 
-When using [Azure DevOps Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) for CI/CD, it is very convenient to leverage the built-in [pipeline variables](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables) for [secrets management](../../readme.md).
-
-However, using pipeline variables for secrets management has a number of disadvantages:
+When using [Azure DevOps Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) for CI/CD, it's convenient to leverage the built-in [pipeline variables](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables) for [secrets management](../../readme.md), but using pipeline variables for secrets management has its disadvantages:
```

Comment:
Should we mention something about the use of variable groups for pipelines (for reading directly from KV too)?

Reply:
added